### ISSUE TITLE
version discrepancy

### ISSUE THREAD

**Issue by kapsakcj** [CFSAN-Biostatistics/shigatyper#12]
CC @rpetit3 

In the StaPH-B docker image for shigatyper v2.0.1, I'm seeing this output:
```
$ docker run staphb/shigatyper:2.0.1 shigatyper --version
ShigaTyper 2.0.0
```
instead of the expected `ShigaTyper 2.0.1`

The docker image is [built with this dockerfile](https://github.com/StaPH-B/docker-builds/blob/master/shigatyper/2.0.1/Dockerfile), which pulls the tag `conda-package-2.0.1` from GitHub and builds/installs shigatyper via running `python3 setup.py install`

When looking at the tag for `conda-package-2.0.1`, the `shigatyper.py` code shows 2.0.0:

https://github.com/CFSAN-Biostatistics/shigatyper/blob/7865ef17cbac4f5fef91d5bb0dc0785b014d7f28/shigatyper/shigatyper.py#L18

Could the code for the command `shigatyper --version` be updated to use the version from `setup.py` instead of what is listed in `shigatyper.py`?

I would give it a try myself and submit a PR but I'm afraid I would not be following the best/proper python practices for versioning & muddy the waters further

**Comment by rpetit3** [comment]
https://github.com/CFSAN-Biostatistics/shigatyper/pull/9#issuecomment-1064684659

Easiest would be fix and rerelease, but changing that line should do it

**Comment by kapsakcj** [comment]
I am all for doing whatever is easiest 😄

**Comment by crashfrog** [comment]
Ok, taking the opportunity (while fixing #13 ) to do version numbering correctly - `shigatyper --version` should return the correct patch version everywhere starting with release 2.0.2 (to come this week.)

**Comment by crashfrog** [comment]
Fixed in 2.0.2.

**Comment by kapsakcj** [comment]
Thanks, can confirm it prints `ShigaTyper 2.0.2` as expected now. Thanks!
